src/user-event/scroll/scroll-to.ts

```diff
@@ -50,31 +63,46 @@ export async function scrollTo(
 
   ensureScrollViewDirection(element, options);
 
+  emitContentSizeChangeEvent(element, options);
```

Comment:
The sequencing of event matches our experiments with RN runtime:
https://github.com/callstack/react-native-testing-library/wiki/ScrollView-Events